netaddr.go

```diff
@@ -426,20 +389,24 @@ func (ip IP) IsMulticast() bool {
 // a nil error are returned. If bits is larger than 32 for an IPv4 address or
 // 128 for an IPv6 address, an error is returned.
 func (ip IP) Prefix(bits uint8) (IPPrefix, error) {
-	if ip.z == z0 {
+	effectiveBits := bits
+	switch ip.z {
+	case z0:
 		return IPPrefix{}, nil
+	case z4:
+		if bits > 32 {
+			return IPPrefix{}, fmt.Errorf("prefix length %d too large for IP address family", bits)
```

Comment:
Since we know it's IPv4, let's say so in the error. Ditto below.

Reply:
Done.